tests/integration/async/test_general.py

```diff
@@ -0,0 +1,102 @@
+import socket
+import pytest
+from time import sleep
+from librouteros.query import Key
+from librouteros import async_connect
+from librouteros.exceptions import LibRouterosError
+
+
+@pytest.mark.asyncio
+async def test_login(routeros_login_async):
+    port, method = routeros_login_async
+    api = None
+    for _ in range(30):
+        try:
+            api = await async_connect(
+                host="127.0.0.1",
+                port=port,
+                username="admin",
+                password="",
+                login_method=method,
+            )
+            break
+        except (LibRouterosError, socket.error, socket.timeout):
+            sleep(1)
+
+    # data = api("/system/identity/print")
+    result = [r async for r in api("/system/identity/print")]
+    assert result[0]["name"] == "Mikrotik-Sachin"
```

Comment:
`Mikrotik-Sachin` where does this come from ?

Reply:
Sorry about that. Was testing something.

Reply:
Done